resources/views/forum/threads/show.blade.php

```diff
@@ -27,24 +27,44 @@
             <a class="btn btn-link btn-block" href="{{ route('forum') }}"><i class="fa fa-arrow-left"></i> Back</a></a>
         </div>
         <div class="col-md-9">
-            <h1>{{ $title }}</h1>
-            <hr>
+            @include('forum.threads._header')
+            <hr class="header-divider">
 
             <div class="panel panel-default">
                 <div class="panel-body forum-content">
                     @md($thread->body())
                 </div>
-
-                @include('forum.threads._footer')
             </div>
 
             @foreach ($thread->replies() as $reply)
                 <div class="panel {{ $thread->isSolutionReply($reply) ? 'panel-success' : 'panel-default' }}">
-                    @if ($thread->isSolutionReply($reply))
-                        <div class="panel-heading">
-                            Solution Reply
+                    <div class="panel-heading thread-info">
+                        <div class="thread-info-avatar">
+                            <img class="img-circle" src="{{ $reply->author()->gratavarUrl(25) }}">
+                        </div>
+
+                        <div class="thread-info-content">
+                            <a href="{{ route('profile', $reply->author()->username()) }}" class="thread-info-link">{{ $reply->author()->name() }}</a> replied
+                            {{ $reply->createdAt()->diffForHumans() }}
+                            @if($thread->isSolutionReply($reply))
```

Comment:
Add a white line above the if statement. Add a space between the `if` and the `(`.